https://github.com/fenrus75/powertop/commit/b9c431aad6cc5383c9571007469eee8e64ec85a6

From b9c431aad6cc5383c9571007469eee8e64ec85a6 Mon Sep 17 00:00:00 2001
From: Manoj Gupta <manojgupta@chromium.org>
Date: Sat, 4 Nov 2017 13:44:04 -0700
Subject: [PATCH] Fix powertop build with libc++.

<ctime> header is not automatically included with libc++.
Add it explicitly to make powertop build with libc++.

This fixes the following errors:
devices/gpu_rapl_device.cpp:35:14: error: use of undeclared identifier
'time'; did you mean 'tie'?
last_time = time(NULL);
                ^~~~
devices/gpu_rapl_device.cpp:45:14: error:use of undeclared identifier
'time'; did you mean 'tie'?
last_time = time(NULL);
                ^~~~

parameters/learn.cpp:161:10: error: use of undeclared identifier
'time'; did you mean 'tie'?
start =	time(NULL);
            ^~~~
---
 src/lib.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lib.h b/src/lib.h
index b64bb0f5433f..6d85eb6e4b40 100644
--- a/src/lib.h
+++ b/src/lib.h
@@ -54,7 +54,7 @@ extern const char *kernel_function(uint64_t address);
 
 
 
-
+#include <ctime>
 #include <string>
 using namespace std;
 
-- 
2.15.1

